src/app/pages/apps/components/installed-apps/installed-apps.component.html

```diff
@@ -44,6 +45,9 @@
               (keydown.enter)="selectApp(app)"
             ></ix-app-row>
           </ng-container>
+          <div *ngIf="!dataSource.length" class="no-apps">
+            {{ 'Kubernetes is not running or does not have installed apps' | translate }}
```

Comment:
Let's keep the old wording.

- When Kubernetes is not running – `Applications are not running`. 
- When no installed apps - `No Applications Installed`

![image](https://user-images.githubusercontent.com/351613/220927737-f7d48aa8-450f-4019-8aa4-d6dde97726d9.png)
![image](https://user-images.githubusercontent.com/351613/220927756-5195eb7c-ce7f-40f2-9b51-dd7b42354be1.png)